dashboard/data/utils.py

```diff
@@ -1,6 +1,18 @@
 import pandas as pd
 
 
+def serialize_df(df: pd.DataFrame) -> bytes:
+    """
+    Serialize a dataframe to bytes
+
+    :param df: dataframe to serialize
+    :return: serialized dataframe
+    """
+    serialized_df = df.reset_index().to_parquet()
```

Comment:
is resetting index needed here?
if abused, we may end up with duplicating index column for a dataframe with thousands of compounds

Reply:
That's a good point 🙏👍 just noticed we get an extra index column that way in the csv file:
![obraz](https://github.com/zuzg/drug-screening/assets/82370491/44539ec4-3081-446e-be88-1eeac4378f50)

I followed the convention we already have, i.e. it's been the case to call `reset_index` and `to_parquet` function, with the first one being indeed potentially troublesome, so removed it, now's ok:
![obraz](https://github.com/zuzg/drug-screening/assets/82370491/741d8b76-fe0b-416d-a384-a1835dfa6df6)